src/book/02-system/03-tools/references.bib

```diff
@@ -0,0 +1,50 @@
+@inproceedings{Goodfellow2014,
+ author = {Goodfellow, Ian and Pouget-Abadie, Jean and Mirza, Mehdi and Xu, Bing and Warde-Farley, David and Ozair, Sherjil and Courville, Aaron and Bengio, Yoshua},
+ booktitle = {Advances in Neural Information Processing Systems 27 (NIPS 2014)},
+ editor = {Z. Ghahramani and M. Welling and C. Cortes and N. Lawrence and K. Q. Weinberger},
+ publisher = {Curran Associates, Inc.},
+ title = {Generative Adversarial Nets},
+ volume = {27},
+ year = {2014},
+ month={December}
+}
+
+@inproceedings{Isola2017,
+  author={Isola, Phillip and Zhu, Jun-Yan and Zhou, Tinghui and Efros, Alexei A.},
+  booktitle={IEEE Conference on Computer Vision and Pattern Recognition (CVPR)}, 
+  title={Image-to-Image Translation with Conditional Adversarial Networks}, 
+  year={2017},
+  pages={5967-5976},
+}
+
+@inproceedings{CycleGAN2017,
+  title={Unpaired Image-to-Image Translation using Cycle-Consistent Adversarial Networks},
+  author={Zhu, Jun-Yan and Park, Taesung and Isola, Phillip and Efros, Alexei A},
+  booktitle={IEEE International Conference on Computer Vision (ICCV)},
+  year={2017},
+  month={October},
+  address={Venice, Italy},
+  publisher = {IEEE},
+}
```

Comment:
I have spent the last hour losing my mind on this but looks like the `booktitle` doesn't show up on this reference unless it either has a `pages` or a `volume` attribute. Not sure if that's a major drama but if you know the relevant pages it would ensure the entire reference shows up 🚀

Reply:
Possibly an issue with the bibtex style, might be good to find a nicer one